src/building/compiler-documenting.md

```diff
@@ -20,8 +20,8 @@ either in whole or individually.
   ```
 
   First,
-  the compiler and rustdoc get built to make sure everything is okay,
-  and then it documents the files.
+  the compiler gets built to make sure rustdoc compiles,
```

Comment:
this is super nitpicky and you can ignore it if you like, but I still feel like "make sure rustdoc compiles" is ambiguous. We're not *checking* that it compiles, we're actually compiling it. I would be less worried about this if it weren't for the fact that `check` is a cargo/x.py subcommand 😅

Maybe it makes more sense to just say "first rustdoc is built," ?

Reply:
glad you are "nitpicky"... was not too happy with it, so thanks for this